packages/s/squirrel/xmake.lua

```diff
@@ -0,0 +1,33 @@
+package("squirrel")
+    set_kind("library")
+    set_homepage("http://www.squirrel-lang.org")
+    set_description("Official repository for the programming language Squirrel")
+    set_license("MIT")
+
+    add_urls("https://github.com/albertodemichelis/squirrel/archive/refs/tags/$(version).tar.gz",
+             "https://github.com/albertodemichelis/squirrel.git")
+
+    add_versions("v3.2", "02805414cfadd5bbb921891d3599b83375a40650abd6404a8ab407dc5e86a996")
+
+    add_deps("cmake")
+
+    on_install(function (package)
+        if package:is_plat("iphoneos") then
+            package:build_addenv("defines", "IOS")
```

Comment:
? defines is not environment vars

Reply:
It's used by cmake tools:
https://github.com/xmake-io/xmake/blob/master/xmake/modules/package/tools/cmake.lua#L176-L185

it seemed like the easiest way to pass a define to CMake (this libs expects `IOS` define on iOS)

Reply:
we can pass cxflags to tools.cmake.

```
cmake.install(package, configs, {cxflags = "-DIOS"})